### ISSUE TITLE
Bug - DataList - truncated content does not trigger a tooltip and is not focusable

### ISSUE THREAD

**Issue by thatblindgeye** [patternfly/patternfly-react#11389]
**Describe the problem**
The truncated content in examples (e.g. [controlling text example](https://www.patternfly.org/components/data-list#controlling-text)) cannot be navigated to via keyboard.

Additionally, a tooltip must trigger for truncated content, otherwise that content gets lost to users. 

See how Alert handles truncation:

https://github.com/patternfly/patternfly-react/blob/a2cb15519172f33cc701fd6b6cc5e7405a6d8a4c/packages/react-core/src/components/Alert/Alert.tsx#L198-L201

**How do you reproduce the problem?**
Go to the above example and notice that no tooltip triggers on mouse hover and that the truncated content cannot be focused.

**Expected behavior**
A tooltip triggers showing all of the text content via mouse hover and keyboard focus.

**Is this issue blocking you?**
List the workaround if there is one.

**Screenshots**
If applicable, add screenshots to help explain the issue.

**What is your environment?**

**What is your product and what release date are you targeting?**

**Any other information?**

**Comment by thatblindgeye** [comment]
This could either be an update to the internal code on our side, or an update to the example code -- most likely may require an internal update, though.
